Ansh

lightnarcissus opened the pull request.

jorgegarcia commented:
Hello @lightnarcissus 

Thanks for this and for including some documentation, good work! The only thing I'm not quite sure is the structure and the location of the Unity project. 

What about creating a new folder inside of tests/ called something like "touchosc_integration" and moving there the .sln and the rest of Unity the files? That way we can keep it neater for the users that are cloning the repository, as the src files are duplicated in the Assets folder of the Unity project. 

Please feel free to arrange it yourself (update the documentation if needed), and I will merge the changes after you are done.

Cheers,
Jorge

lightnarcissus commented:
Moved the scripts and the test scene to a separate touchosc_integration folder, cleaned up the .sln and updated the documentation.

Let me know if you feel any other changes are needed.
Thank you!

jorgegarcia commented:
That's good, thanks. But what I meant is to get the entire Unity project related files moved to tests/touchosc_integration. So, e.g. the solution, Library/, ProjectSettings/ etc all are moved and we only keep the folders docs/, tests/ and src/ in the root. Does it make sense to you?

lightnarcissus commented:
Hey, sorry for the confusion. I've moved all the project files into the touchosc_integration folder now. There should only be /docs, /src and /tests in the root folder now.

jorgegarcia commented:
No worries, thank you :)
